File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
def sol():
    n = int(input())
    m = map(int, input().split())
    answer = 0
    for i in m:
        num = 0
        for j in range(1, i+1):
            if i % j == 0:
                num += 1

        if num == 2:
            answer += 1
    return answer

if __name__ == '__main__':
    print(sol())

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.java
import java.util.Scanner;

public class Main {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int n = Integer.parseInt(sc.nextLine());
    int answer = 0;
    String line = sc.nextLine();
    String[] numbers = line.split(" ");
    for(String number : numbers){
      // 1부터
      int count = 0;
      int num = Integer.parseInt(number);
      for(int i = 1; i <= num; i++){
        if (num % i == 0) {
          count ++;
          if (count > 2){
            break;
          }
        }
      }
      if(count == 2){
        answer ++;
      }
    }

    System.out.println(answer);
    sc.close();
  }
}